build/params_calibnet.go

```diff
@@ -79,8 +79,8 @@ const UpgradeLightningHeight = 489094
 // 2023-04-21T16:00:00Z
 const UpgradeThunderHeight = UpgradeLightningHeight + 3120
 
-// ??????????
-const UpgradeWatermelonHeight = 999999999999999
+// 2023-10-19T13:00:00Z
+const UpgradeWatermelonHeight = 1013200
```

Comment:
I don't think the time here is exactly right? It's 2023-10-19T13:33:00Z, I think?

Reply:
(1013134 would be 2023-10-19T13:00:00Z)

Reply:
I'm just going to update the comment for now and merge -- we can change later if I'm wrong.

Reply:
You are correct, Epoch `1013134` would be 2023-10-19T13:00:00Z! Never trust ChatGPT when doing many follow-up questions about epochs I guess!